crates/iracing-data-api-client/docs/IracingApiResponse.md

```
@@ -0,0 +1,12 @@
+# IracingApiResponse
+
+## Properties
+
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**link** | **String** | A link to the cached data | 
```

markdownlint-cli2 (0.22.0)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)